src/components/autocompleteInput.js

```diff
@@ -112,19 +112,20 @@
     const belowMinimumMessage = useText(validationBelowMinimum);
     const helperTextResolved = useText(helperTextRaw);
     const modelProperty = getProperty(property || '') || {};
-    const labelProperty = getProperty(labelPropertyId) || {};
     const { modelId: propertyModelId, referenceModelId } = modelProperty;
     const { contextModelId } = model;
     const modelId =
       contextModelId || referenceModelId || propertyModelId || model || '';
+    const idProperty = getIdProperty(modelId) || {};
```

Comment:
how can `idProperty` be an empty `{}`?

Reply:
Well, if you don't have a modelId (when you drag it on your canvas), we need to have a fallback.

Reply:
It is the property yes, not the id of the id property